src/tribler/gui/widgets/downloadspage.py

```diff
@@ -109,13 +118,16 @@ def start_loading_downloads(self):
         self.loading_list_item = LoadingListItem(self.window().downloads_list)
         self.window().downloads_list.addTopLevelItem(self.loading_message_widget)
         self.window().downloads_list.setItemWidget(self.loading_message_widget, 2, self.loading_list_item)
-        self.schedule_downloads_timer(now=True)
```

Comment:
It is not necessary to refresh the downloads on the Tribler start because they will be refreshed when the page is shown.